### DISCUSSION TITLE
Would be safe to create a custom table?

### DISCUSSION THREAD

**Discussion by soulsands** [TriliumNext/Trilium#3750] [Q&A] [answered]
Hi zadam! Thanks for your great work bringing people this excellent note app.
now I'm trying build some other features based on internal sql database.  
I created a custom table and want to make sure if is safe to do it. 
Would my table be safe like internel tables?

**Comment by zadam** [comment, marked as answer]
Hi, your custom table won't be deleted or anything like that. One caveat is that it won't get replicated to other instances using the sync protocol.

**Reply by soulsands** [reply]
Thanks for replying!  That would be perfect if my custom table could be synced as well, it would bring a lot of possibilities.
